shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/parameter/impl/EncryptPredicateParameterRewriter.java

```diff
@@ -65,8 +66,17 @@ public void rewrite(final ParameterBuilder parameterBuilder, final SQLStatementC
         String tableName = encryptCondition.getTableName();
         String columnName = encryptCondition.getColumnName();
         return getEncryptRule().findAssistedQueryColumn(tableName, columnName).isPresent()
-                ? getEncryptRule().getEncryptAssistedQueryValues(schemaName, tableName, columnName, originalValues) 
-                        : getEncryptRule().getEncryptValues(schemaName, tableName, columnName, originalValues);
+                ? checkSortable(encryptCondition, getEncryptRule().getEncryptAssistedQueryValues(schemaName, tableName, columnName, originalValues)) 
+                        : checkSortable(encryptCondition, getEncryptRule().getEncryptValues(schemaName, tableName, columnName, originalValues));
+    }
+    
+    private List<Object> checkSortable(final EncryptCondition encryptCondition, final List<Object> values) {
```

Comment:
Hi @cheese8, can we remove return result of this method? Judging from the name of the checkSortable method, it seems that it only checks the values.